services/account-without-juggler/repositories/account/models/Account.ts

```diff
@@ -9,4 +9,19 @@ import {
 export class Account extends Entity {
   static definition = new ModelDefinition('Account', require('./account/model-definition').properties);
   static modelName = 'Account';
+
+  id: string;
+  customerNumber: string;
+  balance: number;
+  branch: string;
+  type: string;
+  avgBalance: number;
+  minimumBalance: number;
+
+  constructor(body?: Object) {
+    super();
+    if (body) {
```

Comment:
it would be nice if `Entity` constructor supported this use case OOTB, so that we could call `super(body)`.